src/constants.js

```diff
@@ -0,0 +1,39 @@
+const { keys } = require('lodash');
+
+// The client names we'll allow in the `{name: lib}` pairing.
+const CLIENT_ALIASES = Object.freeze({
+  pg: 'postgres',
+  postgresql: 'postgres',
+  sqlite: 'sqlite3',
+});
+
+const SUPPORTED_CLIENTS = Object.freeze(
+  [
+    'mssql',
+    'mysql',
+    'mysql2',
+    'oracle',
```

Comment:
I  think that oracle is not actually supported by knex. It is just used by oracledb, which is supported.

Reply:
Thanks, will remove

Reply:
afaik. it is not tested either